### PR TITLE
NUX: disable onboarding signup variation for the desktop app

### PR DESCRIPTION
**Pull request by ramonjd**
This PR disables the new signup flow in the desktop app for now until we can perform further testing.

## Testing

1. Track AB test activation by pasting the following in your console: `localStorage.setItem( 'debug', 'calypso:abtests' );`
2. Enable the onboarding flow: `localStorage.setItem( 'ABTests', '{"improvedOnboarding_20181023" : "onboarding"}' );`
3. Visit `/start` and check to see that the AB test `improvedOnboarding_20181023` has an existing variation of "onboarding"
4. Visit `/start/about?flags=-signup/onboarding-flow` to disable the feature flag. Check to see that the AB test `improvedOnboarding_20181023` has an existing variation of "main"

### CODE REVIEW

File: client/signup/config/flows.js [hunk 1]
```diff
@@ -130,7 +130,9 @@ const Flows = {
 	filterFlowName,
 	filterDestination,
 
-	defaultFlowName: abtest( 'improvedOnboarding' ) || 'main',
+	defaultFlowName: config.isEnabled( 'signup/atomic-store-flow' )
```
Comment (southp) [inline client/signup/config/flows.js]: Shouldn't this be `'signup/onboarding-flow'`? :D

Comment (ramonjd) [reply client/signup/config/flows.js]: > Shouldn't this be 'signup/onboarding-flow'?

Ah, yeah, I was... ah, just testing your eyesight :trollface: 

(Thanks btw)

File: config/desktop.json [hunk 2]
```diff
@@ -97,7 +97,8 @@
 		"rubberband-scroll-disable": false,
 		"settings/security/monitor": true,
 		"settings/theme-setup": false,
-		"signup/social": false,
+        "signup/onboarding-flow": false,
+        "signup/social": false,
```
Comment (southp) [inline config/desktop.json]: The indentation here are 8 spaces which is inconsistent with the other lines. Let's make them consistent.

File: config/development.json [hunk 3]
```diff
@@ -166,6 +166,7 @@
 		"settings/security/monitor": true,
 		"settings/security/monitor/wp-note": true,
 		"settings/theme-setup": true,
+        "signup/onboarding-flow": true,
```
Comment (southp) [inline config/development.json]: Indentation :)

File: config/horizon.json [hunk 4]
```diff
@@ -105,6 +105,7 @@
 		"server-side-rendering": true,
 		"settings/security/monitor": true,
 		"settings/theme-setup": false,
+        "signup/onboarding-flow": true,
```
Comment (southp) [inline config/horizon.json]: Indentation :)

File: config/production.json [hunk 5]
```diff
@@ -115,6 +115,7 @@
 		"server-side-rendering": true,
 		"settings/security/monitor": true,
 		"settings/theme-setup": false,
+        "signup/onboarding-flow": true,
```
Comment (southp) [inline config/production.json]: Well, you know ...

File: config/stage.json [hunk 6]
```diff
@@ -121,6 +121,7 @@
 		"settings/security/monitor": true,
 		"settings/security/monitor/wp-note": true,
 		"settings/theme-setup": false,
+        "signup/onboarding-flow": true,
```
Comment (southp) [inline config/stage.json]: Same

File: config/wpcalypso.json [hunk 7]
```diff
@@ -133,6 +133,7 @@
 		"settings/security/monitor": true,
 		"settings/security/monitor/wp-note": true,
 		"settings/theme-setup": true,
+        "signup/onboarding-flow": true,
```
Comment (southp) [inline config/wpcalypso.json]: You know :)

**Pull request merged**